src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -62,19 +64,34 @@ public class AzureAppConfigurationOptions
         internal TokenCredential Credential { get; private set; }
 
         /// <summary>
-        /// A collection of <see cref="KeyValueSelector"/>.
+        /// Key Value selectors specified by user.
         /// </summary>
         internal IEnumerable<KeyValueSelector> KeyValueSelectors => _kvSelectors;
 
+        /// <summary>
+        /// Feature Flag selectors specified by user.
+        /// </summary>
+        internal IEnumerable<KeyValueSelector> FeatureFlagSelectors => _featureFlagSelectors;
+
+        /// <summary>
+        /// Indicates if <see cref="AzureAppConfigurationRefreshOptions.RegisterAll"/> was called.
+        /// </summary>
+        internal bool RegisterAllEnabled { get; private set; }
+
+        /// <summary>
+        /// Refresh interval for selected key-value collections when <see cref="AzureAppConfigurationRefreshOptions.RegisterAll"/> is called.
+        /// </summary>
+        internal TimeSpan KvCollectionRefreshInterval { get; private set; }
+
         /// <summary>
         /// A collection of <see cref="KeyValueWatcher"/>.
         /// </summary>
-        internal IEnumerable<KeyValueWatcher> ChangeWatchers => _changeWatchers;
+        internal IEnumerable<KeyValueWatcher> KvWatchers => _sentinelKvWatchers;
```

Comment:
This name should also be updated - SentinelKvWatchers / IndividualKvWatchers / something like that?